jung-algorithms/src/test/java/edu/uci/ics/jung/algorithms/shortestpath/TestMinimumSpanningTree.java

```diff
@@ -33,6 +34,16 @@ public void testDAG() {
     assertEquals(graph.edges(), mst.edges());
   }
 
+  public void testSanity() {
```

Comment:
As far as I can tell, if you just replace (in testSimpleTree() and testDAG()) the two parallel calls to check that the node and edge sets are the same with the single check you have here to verify that the _graph_ is actually the same, then this separate test becomes unnecessary.